apps/storefront/.env.template

```diff
@@ -30,3 +30,6 @@ ORYX_STORYBLOK_TOKEN=
 # Used to set a preset name, available options: b2b, b2c.
 # Default preset is b2c (storefront).
 ORYX_PRESET=
+
+# Used to set a feature level, available options: latest, 1_0, 1_1, etc.
+ORYX_FEATURE_LEVEL=
```

Comment:
Maybe `ORYX_FEATURE_VERSION`?

Reply:
`ORYX_FEATURE_VERSION` doesn't sound right, as this is about opt-in to the latest minor.